digiwf-libs/digiwf-spring-security/digiwf-spring-security-core/src/main/java/de/muenchen/oss/digiwf/spring/security/client/ServiceAccountAccessTokenConfiguration.java

```diff
@@ -12,15 +13,13 @@
 import org.springframework.security.oauth2.client.endpoint.DefaultClientCredentialsTokenResponseClient;
 import org.springframework.security.oauth2.client.http.OAuth2ErrorResponseErrorHandler;
 import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
-import org.springframework.security.oauth2.core.OAuth2AccessToken;
 import org.springframework.security.oauth2.core.http.converter.OAuth2AccessTokenResponseHttpMessageConverter;
 import org.springframework.web.client.RestTemplate;
 
-import java.util.function.Supplier;
-
 import static java.util.Arrays.asList;
 
 @Configuration
+@ConditionalOnProperty(value = "digiwf.security.service-account", matchIfMissing = true)
```

Comment:
Is there a reason why it is possible to disable the  ServiceAccountAccessTokenConfiguration?

Reply:
You might have a use case, where the client has no service account configured, because this is not required.